models/domains_tlds.php

```diff
@@ -717,6 +717,28 @@ private function migrateModule($tld, $new_module_id)
             $this->Record->insert('domains_packages', ['tld_id' => $tld->id, 'package_id' => $package_id]);
         }
 
+        // Update welcome email, if migrating from Generic Domains
```

Comment:
I think this is not called when we want it to be.  For example, when we create a TLD with the generic module, and then change it to Namecheap, it will still use the email from the generic module.